lib/IRGen/StructMetadataVisitor.h

```diff
@@ -63,8 +63,11 @@ template <class Impl> class StructMetadataVisitor
 
     // Struct field offsets.
     asImpl().noteStartOfFieldOffsets();
-    for (VarDecl *prop : Target->getStoredProperties())
-      asImpl().addFieldOffset(prop);
+    for (VarDecl *prop : Target->getStoredProperties()) {
+      if (!(prop->getClangDecl() &&
```

Comment:
I'd prefer if this check went through the same function as isExportableField to determine whether the particular VarDecl is exported. It doesn't matter so much on the 6.2 branch, but I can imagine us tuning this predicate in the future, and it would be best to have it in a single place.